### PR TITLE
Adding poro howto

### PR DESCRIPTION
**Pull request by DrSayre**
#### Purpose
Easier to find documentation on using AMS with a PORO

#### Changes
Adds a guide under how to showing how to create a PORO that can be used with AMS.

#### Related GitHub issues
#1521

### CODE REVIEW

File: docs/howto/serialize_poro.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# my_model.rb
+class MyModel
+  alias :read_attribute_for_serialization :send
+  attr_accessor :id, :name, :level
+  
+  def initialize(attributes)
+  @id = attributes[:id]
+  @name = attributes[:name]
+  @level = attributes[:level]
+end
```
Comment (NullVoxPopuli) [inline docs/howto/serialize_poro.md]: can you fix the indentation on these lines?

File: docs/howto/serialize_poro.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[Back to Guides](../README.md)
+
+# How to serialize a Plain-Old Ruby Object (PORO)
+
+When you are first getting started with ActiveModelSerializers, it may seem only `ActiveRecord::Base` objects can be serializable, but pretty much any object can be serializable with ActiveModelSerializers.  Here is an example of a PORO that is serializable:
+```ruby
+# my_model.rb
+class MyModel
+  alias :read_attribute_for_serialization :send
```
Comment (NullVoxPopuli) [inline docs/howto/serialize_poro.md]: nice

File: docs/howto/serialize_poro.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  @name = attributes[:name]
+  @level = attributes[:level]
+end
+  def self.model_name
+    @_model_name ||= ActiveModel::Name.new(self)
+  end
+end
+```
+
+Fortunately, ActiveModelSerializers provides a [`ActiveModelSerializers::Model`](https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model_serializers/model.rb) which you can use in production code that will make your PORO a lot cleaner.  The above code now becomes:
```
Comment (NullVoxPopuli) [inline docs/howto/serialize_poro.md]: :-)

**Comment by NullVoxPopuli** [comment]
hey, can you add a link to your new doc under `HowTo` in this file? https://github.com/rails-api/active_model_serializers/tree/master/docs

and then add a changelog entry to give yourself credit? :-)

**Comment by NullVoxPopuli** [comment]
one last thing! :-)

@DrSayre can you squash your commits?

@bf4 can we enable the squashing feature on github so we don't have to ask to squash?

**Comment by DrSayre** [comment]
@NullVoxPopuli can you tell me how to do that?

**Comment by NullVoxPopuli** [comment]
yup, I have some instructions here: https://github.com/NullVoxPopuli/git-workflow/blob/master/single-repo/README.md#preparing-your-feature--bug-fix-to-be-merged

lemme know if you have any questions

**Comment by DrSayre** [comment]
@NullVoxPopuli let me know if I did it correctly

**Comment by NullVoxPopuli** [comment]
it looks like you did!

**Comment by NullVoxPopuli** [comment]
LGTM! thanks, @DrSayre

**Comment by NullVoxPopuli** [comment]
oh, it looks like I could have squashed. It's after I click merge. oops.

**Comment by DrSayre** [comment]
no worries, glad I could help out!

**Pull request merged**